lambda/managedeni.py

```diff
@@ -142,16 +142,29 @@ def detach_eni(eni_id, attachment_id):
     return True
 
 def eni_assign_tag(eni_id, tag):
-    eni = ec2_resource.NetworkInterface(id=eni_id)
-    tag = eni.create_tags(
-        Tags=[
-            {
-                'Key': 'cluster-eni-id',
-                'Value': tag
-            }
-        ]
-    )
-    pass
+    max_retry = 10
+    retries = 0
+    sleep_interval = 5
+    while True and retries < max_retry:
```

Comment:
Why do we need `True`?

Reply:
I copied this retry logic from `eni_wait_for_creation`. I think we don't need `retries < max_retry` since it will be checked in line 162 but need `True`